packages/devextreme-react/src/core/configuration/utils.ts

```diff
@@ -19,6 +21,21 @@ export function parseOptionName(name: string): IOptionInfo | ICollectionOptionIn
   };
 }
 
+export function buildFullName(node: IConfigNode): string {
```

Comment:
make it more specific and move to the config-node module
```suggestion
export function buildNodeFullName(node: IConfigNode): string {
```